Dockerfile

```diff
@@ -0,0 +1,34 @@
+# 1: Build the exe
+FROM rust:1.56 AS builder
+
+# WORKDIR /usr/src
+
+# RUN USER=root cargo new agora
+
+# COPY Cargo.toml Cargo.lock /usr/src/agora/
+
+WORKDIR /usr/src/agora
+
+COPY . ./
+RUN cargo build --release
+
+FROM ubuntu AS final
+
+ARG APP=/usr/src/app
+ENV APP_USER=agora
```

Comment:
I think this doesn't need to be configurable.

```suggestion
```

Reply:
it's not meant to be configurable, I just wanted to use a variable since its used multiple times in the file.